productized/modules

```diff
@@ -0,0 +1,5 @@
+pom.xml;drools-beliefs,drools-traits,drools-verifier,drools-persistence,drools-examples,drools-examples-api,drools-test-coverage,drools-scenario-simulation,drools-metric,drools-impact-analysis,drools-retediagram,drools-fastutil,kie-maven-plugin,kie-archetypes,jpmml-migration-recipe,drools-drlonyaml-parent
```

Comment:
@mariofusco do you mean module `drools-model-prototype` from drools-model/pom.xml?  This is the only module I could find in the code related to prototypes.

If this is the module you meant, I tried to remove it but the build fails because `org.drools:drools-model-codegen` depends on `org.drools:drools-model-prototype`

Reply:
> If this is the module you meant, I tried to remove it but the build fails because `org.drools:drools-model-codegen` depends on `org.drools:drools-model-prototype`

Ah, you're totally right. I will check if I can decouple them and make that dependency optional, but for now I think it is ok to merge this PR as it is then.